protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -751,10 +760,22 @@ message ListLedgerEntriesRequest {
     optional DateRange date_range = 3;
 }
 
+message ExportLedgerEntriesRequest {
+    optional string path = 1;
+    optional string filename = 2;
```

Comment:
Similar to this https://github.com/vegaprotocol/vega/issues/7029 I don't think we can do it this way. Can we not return the file as a base64 string or is it too big?

Reply:
Yes, I can do it with a base64 filename returned in the response;  without path and filename settings in the request.
Thanks!

Reply:
But if I don't have access to the machine that datanode is running on, how will I get to the file?

Reply:
All right, so in that case by "file as a base64 string" I assume you mean the file content as a base64, is this correct?
In that case the content will be returned in the response and written to a file outside the core part, yes?

Reply:
we should return the file yes, no need to base64 encode it. let's just return the raw byte, and when using the REST endpoint for it. let's add the content-type:text/csv header so browser can response properly.
No need to expose on graphql I suppose?

Reply:
Okay.
Yes, I havent exposed it in graphql